gradio/components/image.py

```diff
@@ -156,7 +157,11 @@ def preprocess(
         else:
             name = "image"
             suffix = "png"
-        im = _Image.open(payload.path)
+
+        if suffix.lower() == "svg":
```

Comment:
I'm worried users will be confused if they set the type to be `numpy` or `PIL` and get a filepath for svg. Might be worth being very clear in the docstring about this but otherwise I think this is a fair compromise.